engines/bops_config/spec/system/application_types_spec.rb

```diff
@@ -108,6 +108,17 @@
 
     expect(page).to have_content("Document tags successfully updated")
 
+    # Set decisions
+    expect(page).to have_selector("h1", text: "Choose decisions")
+    click_button "Continue"
+
+    expect(page).to have_content("Decisions can't be blank")
```

Comment:
Minor detail but do we want a more helpful error message here.
"You must choose at least one decision" etc or maybe ask Laura/Fede